ocs_ci/utility/powernodes.py

```diff
@@ -56,6 +61,30 @@ def verify_machine_is_down(self, node):
         else:
             return True
 
+    def stop_powernodes_machines_powervs(
+        self, powernode_machines, timeout=900, wait=True, force=True
+    ):
+        """
+        Stop PowerNode Machines - PowerVS
+
+        Args:
+            powernode_machines (list): PowerNodes objects
+            timeout (int): time in seconds to wait for node to reach 'not ready' state
+            wait (bool): True if need to wait till the restarted node reaches timeout
+                - for future use
+            force (bool): True for PowerNode ungraceful power off, False for
+                graceful PowerNode shutdown - for future use
+
+        """
+        for pnode in powernode_machines:
+            self.service.stop(pnode, timeout)
+
+        # Wait for an additional 300+60 seconds (for pods to drain)
+        waiting_time = 360
+        logger.info(f"Waiting for {waiting_time} seconds")
+        time.sleep(waiting_time)
+        logger.info(f"Waiting over for {waiting_time} seconds")
```

Comment:
This log message is similar to the one before and makes it seem like we're waiting again. I would make this a bit more clear("Done waiting" or "Continuing...") or simply remove this log statement entirely.

Reply:
Done. I removed the second log parameter.